apps/src/blockly/addons/functionEditor.js

```diff
@@ -27,3 +27,48 @@ export default class FunctionEditor {
   // TODO
   openWithNewFunction() {}
 }
+
+export function procedureFlyoutCallback(workspace) {
```

Comment:
Could this function name describe what it does, rather than how it's used?

Reply:
The equivalent function in core Block is `flyoutCategory`. Renaming to `proceduresFlyoutCategory` to better align.
https://github.com/gonfunko/blockly/blob/4860b4f72f357b6c4e96a65ce2a95a913844bda9/core/procedures.js#L194